v1/v1-api/src/commonMain/kotlin/com/codingfeline/twitter4kt/v1/api/statuses/homeTimeline.kt

```diff
@@ -0,0 +1,64 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.codingfeline.twitter4kt.v1.api.statuses
+
+import com.codingfeline.twitter4kt.core.ApiResult
+import com.codingfeline.twitter4kt.core.apiUrl
+import com.codingfeline.twitter4kt.core.util.Twitter4ktInternalAPI
+import com.codingfeline.twitter4kt.core.util.appendNotNulls
+import com.codingfeline.twitter4kt.v1.api.getInternalListResponse
+import com.codingfeline.twitter4kt.v1.model.status.Tweet
+import io.ktor.http.Parameters
+
+/**
+ * Get's the authenticated user's home timeline.  This will return the full 280 characters
+ * of a tweet in the fullText field rather than the text field
+ *
+ * This is rate limited.
+ *
+ * [Twitter API reference](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/timelines/api-reference/get-statuses-home_timeline)
+ *
+ * @param count Specifies the number of records to retrieve. Must be less than or equal to 200. Defaults to 20. The value of count is best thought of as a limit to the number of tweets to return because suspended or deleted content is removed after the count has been applied.
+ * @param sinceId Returns results with an ID greater than (that is, more recent than) the specified ID. There are limits to the number of Tweets which can be accessed through the API. If the limit of Tweets has occured since the since_id, the since_id will be forced to the oldest ID available.
+ * @param maxId Returns results with an ID less than (that is, older than) or equal to the specified ID.
+ * @param trimUser When set to either `true` , `t` or `1` , the response will include a user object including only the author's ID. Omit this parameter to receive the complete user object.
+ * @param excludeReplies This parameter will prevent replies from appearing in the returned timeline. Using exclude_replies with the count parameter will mean you will receive up-to count Tweets — this is because the count parameter retrieves that many Tweets before filtering out retweets and replies.
+ * @param includeEntities The entities node will not be included when set to false.
+ */
+@OptIn(Twitter4ktInternalAPI::class)
+public suspend fun StatusesApi.homeTimeline(
+    count: Int? = null,
+    sinceId: Long? = null,
+    maxId: Long? = null,
+    trimUser: Boolean? = null,
+    excludeReplies: Boolean? = null,
+    includeEntities: Boolean? = null
+): ApiResult<List<Tweet>> {
+    val parameters = Parameters.build {
+        appendNotNulls(
+            "count" to count,
+            "since_id" to sinceId,
+            "max_id" to maxId,
+            "trim_user" to trimUser,
```

Comment:
Could you remove this as per #21 ?
Sorry for the extra work.

Reply:
Which piece do you want me to remove?

Reply:
trim_user parameter🙏

Reply:
I believe the latest commit resolves this issue